Framework/DataHandling/test/SaveNexusGeometryTest.h

```diff
@@ -0,0 +1,106 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_DATAHANDLING_SAVENEXUSGEOMETRYTEST_H_
+#define MANTID_DATAHANDLING_SAVENEXUSGEOMETRYTEST_H_
+
+#include "MantidAPI/WorkspaceFactory.h"
+#include "MantidAPI/WorkspaceProperty.h"
+#include "MantidDataHandling/SaveNexusGeometry.h"
+#include "MantidTestHelpers/WorkspaceCreationHelper.h"
+
+#include <boost/filesystem.hpp>
+#include <cxxtest/TestSuite.h>
+
+using Mantid::DataHandling::SaveNexusGeometry;
+using MatrixWorkspace_sptr = boost::shared_ptr<Mantid::API::MatrixWorkspace>;
+using Workspace_sptr = boost::shared_ptr<Mantid::API::Workspace>;
+
+class ScopedFileHandle {
```

Comment:
This is copy-paste from `NexusGeometryTest` suite

Reply:
I wasn't able to add an include to where it exists in NexusGeometry/test. possibly because it not in cmakeLists?

Reply:
It will need to go in shared location. Maybe in `TestHelpers`